packages/editor/src/shared/components/PaintNodes/PaintNode.tsx

```diff
@@ -187,9 +187,7 @@ export const PaintNode: React.FC<NodeProps> = ({
       isError={errorNodeNames?.includes(node.name)}
       isMatchedToSearch={isMatchedToSearch}
       isNotSelected={
-        selectedNode?.field
-          ? !compareParserFields(node)(selectedNode.field)
-          : undefined
+        selectedNode?.field ? selectedNode.field.name !== node.name : undefined
```

Comment:
this fixes opacity glitch.

When something was changed in code the selected node had unselected opacity.